pekko-sample-cluster-kubernetes-java/src/main/resources/application.conf

```diff
@@ -33,4 +33,9 @@ pekko.management {
       example-ready = org.apache.pekko.cluster.bootstrap.demo.DemoHealthCheck
     }
   }
+  # Expose port for Kubernetes health check probing.
```

Comment:
Sorry, I forgot the comment should be clarity. I have updated this in the commit, I hope you don't mind.

And then I think this is ready for review.

Reply:
looks good now - thanks